upgrading/manual_upgrades.adoc

```diff
@@ -507,80 +507,38 @@ updated.
 [[updating-policy-definitions]]
 == Updating Policy Definitions
 
-After a cluster upgrade, the default roles
+During a cluster upgrade, the default roles
```

Comment:
During a cluster upgrade and on every restart of any master

Reply:
@enj, can we edit the `policy.json` which restart master use?

Reply:
@enj? ^